.buildkite/src/Command/Dsl.dhall

```diff
@@ -0,0 +1,76 @@
+-- Commands are the individual command steps that CI runs
```

Comment:
Given my feedback on the above, these seems like it is actually `Command/Base.dhall` (after you remove what is currently the "Base"). Dsl is not a good name here because this isn't a Dsl, it's a "component", which is the pattern we are using for structuring "templates". This is just a schema and an interface to build a template.